src/Xrefcheck/Verify.hs

```diff
@@ -113,10 +114,11 @@ data WithReferenceLoc a = WithReferenceLoc
   }
 
```

Comment:
While we're here, let's also delete the `Buildable (VerifyResult e)` instance in line 91, I don't think it's used anywhere.

Reply:
Done